linux/requirements.bsh

```diff
@@ -358,4 +367,29 @@ function docker_compose_version()
 function bash_version()
 {
   "${BASH-bash}" --version | sed -n${sed_flag_rE} 's|[^0-9]+([0-9.()]*).*|\1|;s|\((.*)\)|r\1|;p;q'
-}
\ No newline at end of file
+}
+
+#**
+# .. function:: tar_version_info
+#
+# Sets the vendor and version number of ``tar``.
+#
+# :Parameters: [``TAR``] - The ``tar`` executable what will be called. Can be overwritten to call a different executable. Defaults to ``tar``.
+# :Output: - ``tar_vendor`` - The ``tar`` vendor (BSD or GNU)
+#          - ``tar_version`` - The ``tar`` version number
+#**
+function tar_version_info()
+{
+  local version_info="$("${TAR-tar}" --version)"
+  if [[ "${version_info}" == "tar (GNU tar)"* ]]; then
+    tar_version="$(echo "${version_info}" | awk 'NR==1{print $4}')"
+    tar_vendor="GNU"
+  # macOS
+  elif [[ "${version_info}" == "bsdtar"* ]]; then
+    tar_version="$(echo "${version_info}" | awk '{print $2}')"
+    tar_vendor="BSD"
+  else
+    echo "ERROR unrecognized tar vendor"
+    return 1
```

Comment:
Instead of errorring here, use `echo "${version_info}" | awk 'NR==1{print $NF}'` for the version, and "unknown" for the vendor

This will work, for example, on busy box, and probably any other version of tar we encounter.